### ISSUE TITLE
Cannot run beta on Windows

### ISSUE THREAD

**Issue by steven-gong-artos** [sc-forks/solidity-coverage#458]
Hi,

I got an issue when running coverage in windows:
`truffle run coverage --network coverage`

The coverage report shows 0% coverage for all contracts .sol file:
```
Error: Error: Could not find artifacts for LLibrary from any sources
    at Object.run (C:\Users\abc\AppData\Roaming\npm\node_modules\truffle\build\webpack:\packages\migrate\index.js?abc3:92:1)
    at process._tickCallback (internal/process/next_tick.js:68:7)
Truffle v5.1.0 (core: 5.1.0)
Node v10.16.3
```

All smart contracts under root/contracts/*.sol are annotated by the instrumentation except interfaces, and a coverage report within the root/migration/coverage folder. 

I think this is caused by the truffle test not able to be executed.
However, if I just run:
`truffle test`
All the tests went through successfully.

Are there any configurations I should apply in .solcover.js?
I tried with just 
```
module.exports = {
      norpc: true  
};
```
But doesn't help.

**Comment by cgewecke** [comment]
@steven-gong-artos 

Is there any chance you could link to your project on GitHub if it's public? The only thing in your description that seems like it might be wrong is that the contracts in `root/contracts` should *not* be instrumented. Instead there should be a temp folders called `root/.coverage_contracts` and `root/.coverage_artifacts` which are created and deleted with each coverage run. 

I don't have a Windows computer and it's possible something is going wrong OS wise as well. Could you see if you can `truffle unbox metacoin` and install the coverage plugin and run it on Windows? If not that might be the source of the problem.

**Comment by steven-gong-artos** [comment]
Hi @cgewecke,
Thanks for the advice, I've tried to unbox the metacoin, and installed then run coverage on Windows, got the same issue as below:

> c:/GitHub/metacoin/contracts/ConvertLib.sol
> c:/GitHub/metacoin/contracts/MetaCoin.sol
> c:/GitHub/metacoin/contracts/Migrations.sol
⚠️  This plugin cannot run Truffle's native solidity tests: 1 test(s) will be skipped.


Compiling your contracts...
===========================
> Everything is up to date, there is nothing to compile.


Compiling your contracts...
===========================
> Everything is up to date, there is nothing to compile.

-----------------|-----------|------------|-----------|-----------|-------------------|
File                    |  % Stmts | % Branch |  % Funcs |   % Lines |Uncovered Lines |
-----------------|-----------|------------|-----------|-----------|-------------------|
 contracts\        |             0 |               0 |             0 |            0 |                            |
  ConvertLib.sol |            0 |            100 |            0 |            0 |                        11 |
  MetaCoin.sol   |            0 |               0 |             0 |            0 |      ... 40,43,50,57 |
  Migrations.sol |            0 |               0 |             0 |            0 |     13,22,29,36,39 |
------------------|----------|------------|-----------|----------|--------------------|
All files               |           0 |               0 |             0 |            0 |                             |
------------------|----------|------------|-----------|----------|--------------------|

> Istanbul reports written to ./coverage/ and ./coverage.json
> solidity-coverage cleaning up, shutting down ganache server
Error: Error: Could not find artifacts for Migrations from any sources
    at Object.run (C:\Users\abc\AppData\Roaming\npm\node_modules\truffle\build\webpack:\packages\migrate\index.js?abc3:92:1)
    at process._tickCallback (internal/process/next_tick.js:68:7)
Truffle v5.1.0 (core: 5.1.0)
Node v10.16.3

**Comment by cgewecke** [comment]
@steven-gong-artos Suspect this is WindowsOS and related to the way windows formats paths. Will try to see if I can get basic a Windows CI job running today and identify where things are going wrong.

**Comment by cgewecke** [comment]
Apologies for the delay addressing this. Should be fixed on the newly published latest version: `0.7.0` via #459.

Have added an Windows E2E job which successfully runs Truffle's metacoin tests.

**Comment by steven-gong-artos** [comment]
Thanks, @cgewecke 
Just to confirm that release 0.7.0 fixed my problem.
Happy New Year, and good work! : )
